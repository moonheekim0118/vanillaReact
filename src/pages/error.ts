
const NotFound=()=>{

    const container = document.createElement('div');
    container.className='container';
    const ErrorMessage = document.createElement('h2');
    ErrorMessage.innerText='존재하지 않는 페이지입니다.';
    
    container.appendChild(ErrorMessage);
    return container;
}

export default NotFound();