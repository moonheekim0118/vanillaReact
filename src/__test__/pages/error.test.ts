import "jsdom-global/register";
import error from "../../pages/error";

describe("index page rendering", () => {
    let container;

    it("renders correctly", () => {
        container = error();
    });

    it("matches snapshot", () => {
        expect(container).toMatchSnapshot();
    });
});
