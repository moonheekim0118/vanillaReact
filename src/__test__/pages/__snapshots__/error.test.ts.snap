// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index page rendering matches snapshot 1`] = `
<div
  class="container"
>
  <h2 />
</div>
`;
